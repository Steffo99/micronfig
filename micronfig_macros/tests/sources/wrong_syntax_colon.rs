micronfig::config! {
	GARASAUTO: ,
}

fn main() {
	std::env::set_var("GARASAUTO", "garasauto");
	println!("{:#?}", GARASAUTO());
}
