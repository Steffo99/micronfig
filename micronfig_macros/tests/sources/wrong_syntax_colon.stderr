error: expected one of: `for`, parentheses, `fn`, `unsafe`, `extern`, identifier, `::`, `<`, `dyn`, square brackets, `*`, `&`, `!`, `impl`, `_`, lifetime
 --> tests/sources/wrong_syntax_colon.rs:2:13
  |
2 |     GARASAUTO: ,
  |                ^

error[E0425]: cannot find function, tuple struct or tuple variant `GARASAUTO` in this scope
 --> tests/sources/wrong_syntax_colon.rs:6:19
  |
6 |     println!("{:?}", GARASAUTO())
  |                      ^^^^^^^^^ not found in this scope
